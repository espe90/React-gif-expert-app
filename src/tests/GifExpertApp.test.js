import { shallow } from 'enzyme';
import { GifExpertApp } from "../GifExpertApp";

describe('Testing GifExpertApp Component', () => {
    let wrapper; 
    
    beforeEach(() => {
        wrapper = shallow(<GifExpertApp />);
    })

    test('should create component', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('should show AddCategory component', () => {
        expect(wrapper.find('AddCategory').exists()).toBe(true);
    })

    test('should show GifGrid component', () => {
        expect(wrapper.find('GifGrid').exists()).toBe(true);
    })
    
})
