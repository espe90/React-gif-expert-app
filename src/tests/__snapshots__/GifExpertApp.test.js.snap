// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing GifExpertApp Component should create component 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <ol>
    <GifGrid
      category="One Punch"
      key="One Punch"
    />
  </ol>
</Fragment>
`;
